web/src/components/drawer/record/record-field.tsx

```diff
@@ -129,20 +129,29 @@ export const RecordField: React.FC<RecordFieldProps> = ({
         <TextContent className={`co-resource-item ${size} netobserv-no-child-margin`}>
           <ResourceIcon kind={kind} />
           <Text component={TextVariants.p} className="co-resource-item__resource-name" data-test-id={value}>
-            {value}
+            &nbsp;{value}
           </Text>
         </TextContent>
       )
     );
   };
 
+  const kubeObjContainer = (k: KubeObj) => {
+    const main = kubeObjContent(k.name, k.kind, k.namespace);
+    if (k.showNamespace && k.namespace) {
+      return doubleContainer(main, kindContent('Namespace', k.namespace), false);
+    }
+    return singleContainer(main);
+  };
+
   const kubeObjContent = (value: string | undefined, kind: string | undefined, ns: string | undefined) => {
     // Note: namespace is not mandatory here (e.g. Node objects)
     if (value && kind) {
       return (
         <div data-test={`field-resource-${kind}.${ns}.${value}`} className="force-truncate">
           {resourceIconText(value, kind, ns)}
           <TextContent className="record-field-tooltip netobserv-no-child-margin">
+            {/* Note: THIS IS THE TOOLTIP */}
```

Comment:
If it's not clear here, you can create a `kubeObjTooltip` function returning the whole `TextContent` element. I would avoid having brackets for this comment